docs/cli-reference.md

```diff
@@ -280,19 +280,28 @@ In addition to `.semgrepignore` there are several methods to set up ignore patte
 
 ## Exit codes
 
-`semgrep` may exit with the following exit codes:
+<!-- The exit codes are located in cli/src/semgrep/error.py in open source code of Semgrep repository. -->
 
-- 0: Semgrep ran successfully and found no errors (or did find errors, but the `--error` flag is **not** set).
-- 1: Semgrep ran successfully and found issues in your code (and the `--error` flag is set).
-- 2: Semgrep failed.
+Semgrep can finish with the following exit codes:
+
+- **1**: Semgrep ran successfully and found issues in your code (and the `--error` flag is set).
+- **0**: Semgrep ran successfully and found no errors (or did find errors, but the `--error` flag is **not** set).
+- **2**: Semgrep failed.
+- **4**: Semgrep encountered an invalid pattern in rule schema.
+- **5**: Semgrep configuration is not valid YAML.
+- **7**: At least one rule in configuration is invalid.
+- **8**: Semgrep does not understand specified language.
+- **13**: The API key is invalid.
+- **14**: Semgrep scan failed.
```

Comment:
@brendongo what is the meaning of this exit code? It looks like a duplicate of exit code 2. I could not find it in our codebase outside of its definition (not used anywhere else).